hudi-kafka-connect/demo/setupKafka.sh

```diff
@@ -65,7 +64,7 @@ numBatch=1
 recordValue=0
 recreateTopic="Y"
 
-while getopts ":n:b:tf:k:m:r:o:l:p:s:-:" opt; do
+while getopts ":n:b:t:f:k:m:r:o:l:p:s:-:" opt; do
```

Comment:
`-t` does not expect an argument so we should have `t` instead `t:`.  This change should be reverted.

Reply:
fixed